_posts/2019-01-17-things-to-do-in-nashville.md

```diff
@@ -0,0 +1,37 @@
+In the beginning, Nashville was a small town with a big city vibe, but everyday we are getting bigger and bigger! With such a multitude of options for food, art, music, and nightlife, there is so much that could be put on this list. 
+
+As a start to beginning your Nashville experience, let us suggest just a couple places and give you more resources to find more fun things to do:
+
+### Hot Chicken Restaurants:
+* Hattie B's Hot Chicken: Three locations of just pure bliss in fried deliciously spicy chicken
+* Prince's Hot Chicken: The originator of the hot chicken craze in Nashville
+* Bolton's Hot Chicken & Fish: Another fantastic hot fish and chicken restaurant. Fun fact - it was featured on David Chang's Ugly Delicious!
+
+### Unique Museums and Monuments:
+* The Parthenon: A full scale replica of the Parthenon in Greece
+* The Frist Art Museum:  Van Gogh, Monet, Degas, and Their Times: The Mellon Collection of French Art will be the main exhibit the weekend of our conference.
+* Tootsie's and Robert's Western World: Historic country honky tonks: Tootsie's & Robert's. 
+* Country Music Hall of Fame: A museum dedicated to the history of country music
+* Lane Motor Museum: The largest European collection of cars and motorcycles in the United States.
+
+### Beer, Cider, and Cocktails:
+* Corsair Distillery: A local Nashville distiller of fine spirits
+* Diskin Cider: Crisp apple ciders and kid-friendly hangout space
+* Jackalope Brewery: A local brewery with unique twists on classics with unexpected flavors
+* The Patterson House: An upscale speakeasy bar in the heart of midtown...also, one of Bill's favorites.
+
+### Nashville Favorite Restaurants:
+* Arnold's Country Kitchen: Self described as "Landmark for Southern Soul Food"
+* Peg Leg Porker: Some of the best barbecue in Nashville
+* City House: The James Beard award winning chef and Nashville native, Tandy Wilson's restaurant nestled in historic Germantown
+* Husk: Fine dining in a picturesque Southern mansion overlooking downtown Nashville
+
+### Vegetarian Friendly Restaurants:
+* Avo: A notably delicious vegan restaurant on the west side of Nashville
+* Chaatable & Chauhan's House of Masala & Ale: Both of these restaurants are owned by the venerable Food Network chef, Maneet Chauhan. Chaatable features Indian street food, and Chauhan's  House of Masala and Ale has a higher end feel with a slightly more fusion-like spin.
+* Sunflower Cafe: A widely vegetarian restaurant with quick eats closer to the conference site
+
+### Tea, Desserts, and more:
+* High Garden: Rated one of the best tea houses in the country
+* The Cupcake Collection: A Germantown instution for almost homemade, classic Southern cupcakes
+* Dozen Bakery: A small bakeshop in the Wedgewood Houston area of town with French classic pastries
```

Comment:
Maybe add a little something to the end here? Just a sentence or two?

Reply:
Oh yeah, a callout to tweet at us maybe about what they visit off or on our list?